src/board/board.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Body, Controller, Post, Req, UseGuards } from '@nestjs/common';
+import { BoardService } from './board.service';
+import { CreateBoardDto } from './dto/create-board.dto';
+import { HttpJwtGuard } from '../guards/jwt.guard';
+
+@Controller('board')
+export class BoardController {
+  constructor(private readonly boardService: BoardService) {}
+
+  // Access Token 필요 (작성자 = 토큰의 sub)
+  @UseGuards(HttpJwtGuard)
+  @Post()
+  async create(@Req() req: any, @Body() dto: CreateBoardDto) {
```

Comment:
```suggestion
import { HttpJwtGuard } from '../guards/jwt.guard';
import { Request } from 'express';

@Controller('board')
export class BoardController {
  constructor(private readonly boardService: BoardService) {}

  // Access Token 필요 (작성자 = 토큰의 sub)
  @UseGuards(HttpJwtGuard)
  @Post()
  async create(@Req() req: Request, @Body() dto: CreateBoardDto) {
```

위와 같이 `express`의 `Request` 타입을 지정할 수 있습니다. TypeScript에서 `any` 타입 지정은 지양해야 합니다.

하지만 `req.user`라는 커스텀 속성을 지정했으니 별도 타입을 만들어야겠죠.

그래서 별도 [Custom Decorator](https://github.com/coconutsilo/cs-boilerplate-nestjs-backend/blob/main/src/common/decorators/user.decorator.ts)를 만들어 적용하기도 합니다. 참고 바랍니다.